server/src/searchSNP.js

```diff
@@ -14,6 +14,9 @@
 .lst[ str ]
 	used for byName
 */
+
+import { query_bigbed_by_coord } from './utils.js'
+
 export async function searchSNP(q, genome) {
```

Comment:
looks like this func is now exported by routes/snp.ts. may dedup

Reply:
Done